apps/rings-next/state/common-selectors.ts

```diff
@@ -19,13 +19,21 @@ export const selectRings = createSelector(
             const wearerRel = rels.find(r => r.data?.relationshipAddBody?.type === RelationshipTypes.Wearer);
             const owner = usersById[ownerRel?.data?.relationshipAddBody?.target?.fid];
             const wearer = usersById[wearerRel?.data?.relationshipAddBody?.target?.fid];
-            const stones = stonesByRingId[ringId] || [];
+            const stones = (stonesByRingId[ringId] || []) as Array<Message>;
+
+            // TODO: figure out why the tags are returned from the API with messed up `name` field values
+            //    This issue gets in the way of us matching tags to proper Ring fields.
```

Comment:
The main issue getting in the way of stone values being properly selected in the table

Reply:
Addressed in later commit. There was some special case in the httpServer endpoints that converted object key values that were called "name" from base64 to string